source/_patterns/02-molecules/card/_card.scss

```diff
@@ -137,10 +301,38 @@ $_config: map-merge-by-keys($_config_schemes, base, $_config_schemes, $scheme);
 .usa-card__brow {
   @include u-margin-bottom(1);
   @include u-order(0);
-  @include u-color(map-get($_config, brow-color));
   letter-spacing: 0.09px;
   @include typeset(body, map-get($_config, brow-size), $theme-body-line-height);
   font-weight: map-get($_config, brow-weight);
+
+  .usa-card__brow-color,
+  .usa-card__brow-color > div:first-child {
+    @include u-display(inline);
+    @include u-bg(map-get($_config, brow-bgcolor));
+    @include u-color(map-get($_config, brow-color));
+    @include u-margin-left("neg-3");
+    @include u-padding(1);
+    padding-left: 26px;
+  }
+}
+
+.usa-card__icon {
+  @include u-float("right");
+  @include u-color(map-get($_config, brow-icon-color));
+
+  .icon {
+    width: 32px;
+    height: 24px;
+  }
+}
+
+.usa-card__icon-no-brow {
+  @include u-color(map-get($_config, brow-icon-color));
+
+  .icon {
+    width: 32px;
+    height: 24px;
+  }
 }
 
 .usa-card__footer {
```

Comment:
I'm not seeing this one.  What color are you seeing?

Reply:
Oh wait. If you rebased with the latest, do you see blue arrows now which I haven't tried yet? I may have fixed the color problem when I adjusted the uswds-*.scss values for pro. This might be moot once you PR with master.